packages/datadog-instrumentations/src/cypress.js

```diff
@@ -5,4 +5,4 @@ const { addHook } = require('./helpers/instrument')
 addHook({
   name: 'cypress',
   versions: ['>=6.7.0']
-})
+}, lib => lib)
```

Comment:
this is producing a log when logs are active. It's no big deal but it's an easy fix